.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: weekly
+    ignore:
+      - update-types: ["version-update:semver-patch", "version-update:semver-minor"]
```

Comment:
I just integrated this in an internal repo, and used to get an implicit `dependabot config OK` CI run way back in the day, but it's not there anymore.

Unfortunately now after merging it, the dependabot view complains:

```
The property '#/updates/0/ignore/0' did not contain a required property of 'dependency-name'
```

You'll have to fix the same here, and it's [documented](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore):

> You can combine this with `dependency-name: "*"` to ignore particular `update-types` for all dependencies.